.github/workflows/linters.yml

```diff
@@ -0,0 +1,70 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍